code/game/machinery/hologram.dm

```diff
@@ -51,6 +50,7 @@ var/const/HOLOPAD_MODE = RANGE_BASED
 	var/mob/living/caller_id
 	var/obj/machinery/hologram/holopad/sourcepad
 	var/obj/machinery/hologram/holopad/targetpad

```

Comment:
why is this an object instead of just a simple `var/forced`?

Reply:
because I just copied it from the other ones and they're objects. Will change.